respa_admin/templates/respa_admin/forms/_form_errors.html

```diff
@@ -1,32 +1,9 @@
 {% load i18n %}
-
-{% if form.errors or resource_image_formset.errors or period_formset_with_days.errors or resource_accessibility_formset.errors %}
+{% if errors %}
     <div class="alert alert-danger">
-        <strong>{% trans "Tarkista seuraavat virheet" %}:</strong>
+        <strong>{% trans "Check the following errors" %}:</strong>
         <ul>
-            {% for error in form.errors %}
-                <li>
-                    {{ error }}
-                </li>
-            {% endfor %}
-
-            {% for error in resource_image_formset.errors %}
-                <li>
-                    {{ error }}
-                </li>
-            {% endfor %}
-
-            {% for error in period_formset_with_days.errors %}
-                <li>
-                    {{ error }}
-                </li>
-            {% endfor %}
-
-            {% for error in resource_accessibility_formset.errors %}
-                <li>
-                    {{ error }}
-                </li>
-            {% endfor %}
+            {{ errors|unordered_list }}
```

Comment:
Django filter `unordered_list` will render the list automatically, no need to add each `<li>` element. The tag should return a flattened list of errors from all the included forms and formsets, so we shouldn't need to specify each formset in this code.

Reply:
Nice cleanup! 👏